### PR TITLE
Enable GMC e2e on Gaudi

### PR DESCRIPTION
**Pull request by daisy-ycguo**
## Description

Enable GMC e2e on Gaudi

## Issues

Partially closes issue #118 

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

N/A

## Tests

CI Test

### CODE REVIEW

**Review by zhlsunshine — APPROVED** [review]
Thanks for adding the gaudi e2e test, LGTM!

File: microservices-connector/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# genai-microservices-connector(GMC)
+# genai-microservices-connector(GMC)T
```
Comment (KfreeZ) [inline microservices-connector/README.md:1]: the T reminder :)

Comment (daisy-ycguo) [reply microservices-connector/README.md:1]: will remove after testing.

**Review by KfreeZ — APPROVED** [review]

**Review by zhlsunshine — APPROVED** [review]
LGTM

**Pull request merged**